project/merge_sort.py

```diff
@@ -1,21 +1,57 @@
 def merge(arrA, arrB):
-    print(arrA)
+    # we get the length of the new array
+    new_length = len(arrA) + len(arrB)
+    # we create a new array with zeros in it
+    merged_arr = [0] * new_length
```

Comment:
very nice! this habit of creating the array beforehand with its proper length and the values set to 0 will do you great in C.

However, just so you know: in Python, that's not necessary.  See code below
arr = [0, 1, 2]
arr.append('new item appended to array')
print(arr)   #prints [0, 1, 2, 'new item appended to array']